sdks/go/pkg/beam/combine.go

```diff
@@ -23,14 +23,14 @@ import (
 
 // Combine inserts a global Combine transform into the pipeline. It
 // expects a PCollection<T> as input where T is a concrete type.
-func Combine(s Scope, combinefn interface{}, col PCollection, opts ...Option) PCollection {
-	return Must(TryCombine(s, combinefn, col, opts...))
+func Combine(s Scope, combinefn interface{}, col PCollection) PCollection {
```

Comment:
We're extremely confident that Combine will never need to take options? It seems we're closing a big door here.